contributing/code_of_conduct/enforcement_team.rst

```diff
@@ -0,0 +1,58 @@
+Enforcement team
+================
+
+Our Pledge
+----------
+
+In the interest of fostering an open and welcoming environment, the enforcement team
+pledge to ensure that the spirit of the :doc:`Code of Conduct </contributing/code_of_conduct/index>`
+is respected. Our main priority is to ensure the safety of our community members.
+The second goal is to help educate the community as a whole to be aware of the CoC
+and how to help implement its spirit throughout the community. In case these goals
+conflict, we will prioritize safety of community members over all other goals.
+
+If you think there is or has been a violation to the code of conduct please contact
+enforcement team or if you prefer contact only individual members of the enforcement team.
+
+Members
+-------
+
+Here are all the members of the Code of Conduct enforcement team. You can contact
+any of them by sending an email or tweet. You could also contact all of them at once
+by emailing **coc@symfony.com**.
+
+Lukas Kahwe Smith
+`````````````````
```

Comment:
Minor details: we use `~~~~~~~~~~~~~~` for the third level heading in Symfony Docs.